tools/build/redo

```diff
@@ -58,7 +58,7 @@ def doComponentSequence(props, args, components):
         if not component:
             if args.yaml:
                 file = c if c.endswith('.yml') else '%s.yml' % c
-                component = makeComponent('custom deployment', 'deploying using %s' % file, yaml = file)
```

Comment:
stashed but not related to this comment - general tweaks to `redo` script to support custom YAML files and added gocli build.